### DISCUSSION TITLE
No sound in HDMI under Ubuntu mate Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by pkeglevich** [HenningThiemann/docker-chromium-armhf#24] [General]
Hi;

Very smart this solution for working around the limitations of lacking DRM in Raspberry Pi 4 with Ubuntu Mate 64 bits! It works in Chromium excepts that the Crommium in docker.io have no sound at all when Ubuntu/RPI4 is connected in HDMI (even in Youtube). Outside the docker.io the sound works perfectly both in Chromium and Firefox but I don't have Widevine :(. Any suggestions on how to solve this issue which I believe being of configuration?

**Comment by dupontk67** [comment]
Hello world,

Thanks @HenningThiemann for this great work. Everything is working except sound.
Any idea of how to resolve the sound issue?

Best regards :)
_______
My config : Raspberry Pi 4 with Debian GNU/Linux 11 KDE / Plasma 64 bits 4Go RAM
